src/Select.js

```diff
@@ -742,7 +742,11 @@ class Select extends React.Component {
 
 	renderValue (valueArray, isOpen) {
 		if (!valueArray.length) {
-			return !this.state.inputValue ? <div className="Select-placeholder">{this.props.placeholder}</div> : null;
+			if (this.props.type === 'unique') {
+				return !this.state.inputValue ? <div className="Select-placeholder">No Panel Selected</div> : null;
```

Comment:
you can change this line and the next for 
`!this.state.inputValue && <div className="Select-placeholder">No Panel Selected</div>`

Reply:
I think this is the same 
```
  if (!valueArray.length) {
      if (this.state.inputValue) {
        return null;
      }
      return  <div className="Select-placeholder">{this.props.type === 'unique' ? 'No Panel Selected' : this.props.placeholder}</div>;
  }
```
or maybe we should just pass the `placeholder` prop.

Reply:
I think that we should pass a `placeholder` property rather than hardcode that phrase, 
because that string is just for the new changes in DPP and if the string change then we need to do another release of this library.